File: Programmers/level3/level3_숫자-게임.cpp
#include <string>
#include <vector>
#include <algorithm>

//프로그래머스 level3_숫자-게임

using namespace std;

int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    sort(A.begin(),A.end());
    sort(B.begin(),B.end());
    
    int a=0;
    for(int b=0;b<B.size();b++){
        if(A[a]<B[b]){
            a++;
            answer++;
        }
    }
    
    return answer;
}

File: Programmers/level3/level3_숫자-게임.java
import java.util.*;

class Solution {
    public int solution(int[] A, int[] B) {
        int answer = 0;
        
        Arrays.sort(A);
        Arrays.sort(B);
        
        int idx = 0;
        for(int i=0;i<B.length;i++){
            if(A[idx]<B[i]){
                idx++;
                answer++;
            }
        }
        
        return answer;
    }
}